### PR TITLE
Preview 13 Ready for update

### PR DESCRIPTION
**Pull request by ajhunyady**

### CODE REVIEW

**Comment by ajhunyady** [comment]
@crajcan can you please help resolve the conflicts in the CI file?

File: dataflows-inline/bank-processing/README.MD [hunk 1]
```diff
@@ -56,17 +56,12 @@ Use `fluvio consume` command to read from multiple streams in parallel:
 
 #### 2.1 Add Account
 
-Use fluvio produce to publish `add-account` events to the `admin-events` topic :
+Use fluvio produce to publish `add-account` events for `duncan` and `lucy` to the `admin-events` topic :
 
 Add `duncan`:
```
Comment (crajcan) [inline dataflows-inline/bank-processing/README.MD]: could probably delete this `Add duncan:` line too

File: dataflows-inline/car-processing/README.MD [hunk 2]
```diff
@@ -162,7 +162,10 @@ The window is flushed to the `car-colors` topic every 30 seconds.
 Watermark tracks the cutoff timestamp for tumbling windows.
 
 ```bash
->> show state count-by-color/count-by-color/watermark
+show state count-by-color/count-by-color/watermark
+```
+
+```bash
  Key      Value                     Window
  default  2024-03-15T13:44:30.043Z  *
```
Comment (crajcan) [inline dataflows-inline/car-processing/README.MD:170]: I don't know if it matters but in a similar show state example in `dataflows-inline/split-sentence/README.MD` below you removed the output of the show state command from the README

File: primitives/key-value/chained/README.MD [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# Chained Key/Value Example
+
+This example demonstrates the engine behavior for passing keys between while chaining operators.
```
Comment (crajcan) [inline primitives/key-value/chained/README.MD]:
```suggestion
This example demonstrates the engine behavior for passing keys between chained operators. 
```

File: primitives/key-value/chained/README.MD [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+fluvio consume user-output -Bd -k
+```
+
+```bash
+[child] {"age":10,"id":"1001","location":"New York","name":"Alice"}
+[child] {"age":17,"id":"1002","location":"New York","name":"Bob"}
+[adult] {"age":24,"id":"2003","location":"California","name":"Charlie"}
+```
+
+The existing `key` was into the `value` and a new `key` was assigned.
```
Comment (crajcan) [inline primitives/key-value/chained/README.MD]:
```suggestion
The existing `key` was merged into the `value` and a new `key` was assigned.
```

File: primitives/key-value/input/README.MD [hunk 5]
```diff
@@ -0,0 +1,82 @@
+# Input Key/Value Example
+
+This is an example of consuming records with `key` and `value` from a fluvial topic. We've defined a function that reads key-value records from a topic, collapses them into a single value, and sends the result to a new topic.
```
Comment (crajcan) [inline primitives/key-value/input/README.MD]:
```suggestion
This is an example of sourcing records with `key` and `value` from a fluvio topic. We've defined a function that reads key-value records from a topic, collapses them into a single value, and sends the result to a new topic.
```

File: primitives/key-value/input/README.MD [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+* See the [dataflow.yaml](./dataflow.yaml) to see how we've configured the dataflow.
+* Make sure to [Install SDF and start a Fluvio cluster].
```
Comment (crajcan) [inline primitives/key-value/input/README.MD:6]: We'll need updated urls here

```suggestion
* Make sure to [Install SDF](---url---) and [start a Fluvio cluster](---url---).
```

File: primitives/key-value/output/README.MD [hunk 7]
```diff
@@ -0,0 +1,81 @@
+# Output Key/Value Example
+
+This is an example of producing records with a `key` and a `value` for a fluvial topic. We've defined a function that reads records from a topic, divides them into a key and a value, and sends the result to a new topic.
```
Comment (crajcan) [inline primitives/key-value/output/README.MD]:
```suggestion
This is an example of sinking records with a `key` and a `value` into a fluvio topic. We've defined a function that reads records from a topic, divides them into a key and a value, and sends the result to a new topic.
```

File: primitives/regex/README.MD [hunk 8]
```diff
@@ -39,13 +47,15 @@ fluvio consume gen-z -Bd
 {"name": "Bob", "born": 2011}
 ```
 
+**Note:** the regex filter removes people born outside of `gen-z ` age group.
```
Comment (crajcan) [inline primitives/regex/README.MD]:
```suggestion
**Note:** the regex filter removes people born outside of `gen-z` age group.
```

File: dataflows-inline/bank-processing/README.MD (hunk 1 above)
Comment (ajhunyady) [reply dataflows-inline/bank-processing/README.MD]: good catch.

File: primitives/key-value/chained/README.MD (hunk 3 above)
Comment (ajhunyady) [reply primitives/key-value/chained/README.MD]: Nice rewording. Thanks.

File: primitives/key-value/input/README.MD (hunk 5 above)
Comment (ajhunyady) [reply primitives/key-value/input/README.MD]: good catch.

File: dataflows-inline/car-processing/README.MD (hunk 2 above)
Comment (ajhunyady) [reply dataflows-inline/car-processing/README.MD:170]: I figured it's too long and the display is repetitive. We can bring back if needed.

**Review by crajcan — APPROVED** [review]

**Pull request merged**